realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -76,7 +76,7 @@
      * @return {@link RealmQuery} object. After building the query call one of the {@code find*} methods
      * to run it.
      */
-    public static <E extends RealmModel> RealmQuery<E> createQuery(Realm realm, Class<E> clazz) {
```

Comment:
Are you sure these can just be hidden? Weren't they part of public API?

https://realm.io/docs/java/latest/api/io/realm/RealmQuery.html#createQueryFromResult-io.realm.RealmResults-

Reply:
I think that its being public is mistake.

What do you think? @realm/java

Reply:
I tend to agree. I believe they were public because the original migration/dynamic API had a lot of moving parts and at some point the dynamic parts lived in their own package.

It is a little bad form to just remove them without warning, but all of these should be accessible using normal `realm.where()` methods and the like instead, so probably not a big deal.

As long as it is mentioned in the changelog removing from public is probably fine.

Reply:
Added changelog entry for these.